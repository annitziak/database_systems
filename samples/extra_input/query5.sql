SELECT Student.A FROM Student WHERE Student.A>=3 AND Student.B>=100 AND Student.C>51;
