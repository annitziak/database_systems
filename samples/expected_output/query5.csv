1, 200, 50, 33, 1, 103, 92
2, 200, 200, 44, 2, 101, 12
3, 100, 105, 44, 3, 102, 52
4, 100, 50, 11, 4, 104, 27
1, 200, 50, 33, 1, 102, 82
1, 200, 50, 33, 1, 101, 75
